tests/main/proxy-no-core/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Ensure that the core.proxy.* settings are honored without core
+
+# only needs a test on classic
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
+
+restore: |
+    iptables -D OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
+    iptables -D OUTPUT -j REJECT -p tcp --dport http --reject-with tcp-reset
+    iptables -D OUTPUT -j REJECT -p tcp --dport https --reject-with tcp-reset
+
+execute: |
+    if ! command -v python3; then
+       echo "SKIP: need python3"
+       exit 0
+    fi
+    #shellcheck source=tests/lib/systemd.sh
+    . "$TESTSLIB"/systemd.sh
+    systemctl stop snapd.service snapd.socket
```

Comment:
AFAIK every time I use this order socket is stopped last and system warns me that snapd can be activated by the socket.